### PR TITLE
text

### PR DESCRIPTION
**Pull request by stort4444**

### CODE REVIEW

**Comment by stort4444** [comment]
comment

**Comment by stort4444** [comment]
UItest 🤖

**Comment by stort4444** [comment]
UItest 🤖

**Comment by stort4444** [comment]
UItest 🤖

**Comment by stort4444** [comment]
UItest 🤖

File: testfile [hunk 1]
```diff
@@ -1,2 +1,3 @@
 lol
 kek
```
Comment (stort4444) [inline testfile]: Comment in code

Comment (stort4444) [reply testfile]: Another comment in code

Comment (stort4444) [reply testfile]: Comment for bot 🤖

Comment (stort4444) [reply testfile]: 🤖

**Comment by stort4444** [comment]
New review comment

**Comment by StanislavKrishu** [comment]
🤖

File: testfile (hunk 1 above)
Comment (stort4444) [reply testfile]: New incode comment 🤖

**Comment by stort4444** [comment]
🤖 Comment 🤖

File: testfile (hunk 1 above)
Comment (stort4444) [reply testfile]: 🤖 Up 🤖

**Comment by stort4444** [comment]
Review 🤖

File: testfile (hunk 1 above)
Comment (StanislavKrishu) [reply testfile]: 🤖 WoW 🤖

Comment (StanislavKrishu) [reply testfile]: 🤖 run ui test

Comment (stort4444) [reply testfile]: New 🤖

Comment (stort4444) [reply testfile]: Comment in code 🤖

Comment (stort4444) [reply testfile]: Comment in code 🤖

Comment (stort4444) [reply testfile]: 🤖🤖 Incode

Comment (stort4444) [reply testfile]: 🤖🤖🤖

**Comment by stort4444** [comment]
Leave a comment 🤖

File: testfile (hunk 1 above)
Comment (stort4444) [reply testfile]: run ui tests 🤖

**Comment by stort4444** [comment]
Leave a comment

**Comment by stort4444** [comment]
Leave a comment 🤖

**Comment by stort4444** [comment]
Leave a comment 🤖

**Comment by stort4444** [comment]
LEave a comment 2 🤖

**Comment by stort4444** [comment]
Leave 3 🤖

**Comment by stort4444** [comment]
LEave 🤖

**Comment by stort4444** [comment]
run ui test

**Pull request closed without merging**